service/src/main/java/greencity/service/EmailServiceImpl.java

```diff
@@ -224,7 +228,7 @@ private void changeLocale(String language) {
                 Locale.setDefault(Locale.ENGLISH);
                 break;
             default:
-                throw new IllegalStateException("Unexpected value: " + language);
+                throw new LanguageNotFoundException("Unexpected language: " + language);
         }
```

Comment:
This method should be refactored since Locale sets for the whole VM whenever this method invokes.
etc: LanguageValidator.validate()